### DISCUSSION TITLE
loading Always  in full screen and does not respect the container

### DISCUSSION THREAD

**Discussion by coreqnq** [Happy-Coding-Clans/vue-easytable#491] [General]
<ve-table
              id="loading-container"
              style="width:100%;height:250px;background-color:#2980b9;"
              :columns="columns"
              :table-data="tableData"></ve-table>
            <div class="table-pagination">
              <ve-pagination
                :total="totalCount"
                :page-index="pageIndex"
                :page-size="pageSize"
                :columns="columns"
                @on-page-number-change="pageNumberChange"
                @on-page-size-change="pageSizeChange"
              />
            </div>
 
 .
 .

  data () {
    return {
      loadingInstance: this.$veLoading({
        target:  document.querySelector('#loading-container'),
        fullscreen: false,
        name: 'wave'
      }),

![imagen](https://user-images.githubusercontent.com/85887120/175109418-3736bab8-a35f-49a2-ad36-7192c078fbe9.png)

**Comment by coreqnq** [comment]
l try to put in another container div.id=loading-container but doesn't work

**Comment by huangshuwei** [comment]
> data () {
return {
loadingInstance: this.$veLoading({
target: document.querySelector('#loading-container'),
fullscreen: false,
name: 'wave'
}),

`loadingInstance` needs to be set after rendering，For example:
```
 mounted() {
             this.loadingInstance = this.$veLoading({
                target: document.querySelector("#loading-container"),
                // 等同于
                // target:"#loading-container"
                name: "wave",
            });
        },
```
